arangod/RocksDBEngine/RocksDBMetaCollection.cpp

```diff
@@ -1661,63 +1659,37 @@ Result RocksDBMetaCollection::applyUpdatesForTransaction(
 
 /// @brief lock a collection, with a timeout
 ErrorCode RocksDBMetaCollection::doLock(double timeout, AccessMode::Type mode) {
-  uint64_t waitTime = 0;  // indicates that time is uninitialized
-  double startTime = 0.0;
-
   // user read operations don't require any lock in RocksDB, so we won't get
   // here. user write operations will acquire the R/W lock in read mode, and
   // user exclusive operations will acquire the R/W lock in write mode.
   TRI_ASSERT(mode == AccessMode::Type::READ || mode == AccessMode::Type::WRITE);
+  auto tous = std::chrono::duration_cast<std::chrono::microseconds>(
```

Comment:
Can we please rename this to `timeout_us` or just `timeout`? I know the first one does not match our styleguide, but `tous` sounds really nonsensical to me (is this french?).